docs/insomnia/get-started-with-documents.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: article-detail
+title: Get Started with Documents
+category: "API Design"
+category-url: api-design
+---
+
+Design Documents (also called Documents) hold specifications, API requests, and tests. They also offer the capability to sync with Git.
+
+If you're looking for information about Collections instead (which use Version Control Sync), see [Request Collections](/insomnia/request-collections).
+
+For this guide, create an empty Document from your Dashboard. We've named our Document "petstore" as our example uses the Pet Store API.
+
+## Tab Overview
+
+We're going to start in the Design tab. Then we'll walk through each tab in order from left to right.
+
+![Documents have three tabs: Design, Debug, and Test.](/assets/images/three-document-tabs.png)
+
+_Documents have functionality separated into three tabs: Design, Debug, and Test._
+
+Tab functionality is as follows:
+
+* [**Design**](#design-tab): Import and edit specifications, see API overview content, and preview a rendering of the API.
+* [**Debug**](#debug-tab): Create, edit, and send individual requests. Organize your requests into folders. View request responses.
+* [**Test**](#test-tab): Create test suites and individual tests for requests. Run tests individually or as a group. View test results.
+
+## Design Tab
+
+The Design tab is the first of three top-level tabs in Documents. Import your specification on this tab.
+
+### Add Your Specification
+
+We're going to import a sample file that's built into Insomnia. Click on **start from an example** (above the **Import** button), which will populate the sample Pet Store API.
```

Comment:
I think this is only visible on first launch (not any design doc you make (yet))? We might want to add a link to the examples here: https://github.com/OAI/OpenAPI-Specification/tree/main/examples/v3.0 as an alternative

Reply:
Oh, just saw we have the Alternative Option below. It might be good to call out earlier, particularly if "start from an example" may not be visible always.

Reply:
From my testing, I could click "start from an example" in any new Document I create. Have you had a different experience?

Reply:
Oh, then I think this is great. I was under the impression that for now this was only for the first launch experience.